icicle/src/msm/tests/msm_test.cu

```diff
@@ -170,65 +195,93 @@ int main()
     0,      // mempool
   };
   msm::MSMConfig config = {
-    ctx,   // DeviceContext
-    0,     // points_size
-    1,     // precompute_factor
-    0,     // c
-    0,     // bitsize
-    10,    // large_bucket_factor
-    1,     // batch_size
-    false, // are_scalars_on_device
-    false, // are_scalars_montgomery_form
-    false, // are_points_on_device
-    false, // are_points_montgomery_form
-    true,  // are_results_on_device
-    false, // is_big_triangle
-    true,  // is_async
+    ctx,            // DeviceContext
+    N,              // points_size
+    precomp_factor, // precompute_factor
+    user_c,         // c
+    0,              // bitsize
+    10,             // large_bucket_factor
+    batch_size,     // batch_size
+    false,          // are_scalars_on_device
```

Comment:
why don't you use scalar_d if you already copied them to device? intentional? if yes, no need to allocate scalars_d and copy.

Reply:
It's just a test file, I copy to have all options available, I don't use scalar_d because in zprize you need to include scalar transfer time